### PR TITLE
Remove Caching Test Server from Signal Test

### PR DESCRIPTION
**Pull request by malnick**
Leverages new test flag which dumps to STDOUT so we can remove the usage of caching server.

### CODE REVIEW

**Comment by cmaloney** [comment]
This seems to only update ref_origin but not actually change what is built.

**Comment by malnick** [comment]
@spahl @cmaloney This is ready for review. Works on Vagrant!

**Comment by spahl** [comment]
:ship:

File: packages/dcos-integration-test/extra/integration_test.py [hunk 1]
```diff
@@ -1610,87 +1611,40 @@ def test_3dt_report(cluster):
...
-    print('CACHE SERVER STATUS:\n{}'.format(r.status_code))
+    dcos_version = os.getenv("DCOS_VERSION", "")
+    signal_config = open('/opt/mesosphere/etc/dcos-signal-config.json', 'r')
+    signal_config_data = json.loads(signal_config.read())
+    customer_key = signal_config_data.get('customer_key', '')
+    cluster_id_file = open('/var/lib/dcos/cluster-id')
+    cluster_id = cluster_id_file.read().strip()
 
-    r_data = json.loads(r.json())
+    print("Version: ", dcos_version)
```
Comment (cmaloney) [inline packages/dcos-integration-test/extra/integration_test.py]: we shouldn't be printing here...

**Pull request merged**